### PR TITLE
Patch WriteFile to support bootconfig

### PR DESCRIPTION
**Pull request by vignesh-goutham**
*Description of changes:*
Patching Tink writefile action to support bootconfig. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/tinkerbell/hub/patches/0001-Add-Bootconfig-support-to-WriteFile-action.patch [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+ 	}
+ 
++	if bootconfig != "" {
++		// Write the input bootconfig to file to serve as input to the tool
++		inputFilePath := "/userInputBootConfig"
++		err := ioutil.WriteFile(inputFilePath, []byte(bootconfig), fileMode)
++		if err != nil {
++			log.Fatalf("Could not write file %s: %v", filePath, err)
++		}
++		os.Remove(inputFilePath)
```
Comment (pokearu) [inline projects/tinkerbell/hub/patches/0001-Add-Bootconfig-support-to-WriteFile-action.patch:108]: Are we writing and immediately removing the file?

Comment (vignesh-goutham) [reply projects/tinkerbell/hub/patches/0001-Add-Bootconfig-support-to-WriteFile-action.patch:108]: good call, thats not supposed to be there.

**Review by pokearu — APPROVED** [review]
/lgtm

**Pull request merged**